docs/guides/testing-feature-flags.md

```diff
@@ -0,0 +1,33 @@
+# Testing Feature Flags
+
+## Testing in Staging or Local Environments
+
+Eppo supports separation of feature flags and their allocations between environments. Create an Eppo environment specifically for Staging or QA following this [documentation](/feature-flagging/#environments). Environments are separated by [SDK keys](/sdks/api-keys), and no unique key will be shared across Eppo Environments. Once an Environment is created, create an SDK for that Environment.
+
+After a flag has been created, click **Switch Environment** on the flag page and select the testing Environment the flag should run in.
+
+![Screenshot of list of Environments](/img/guides/testing-feature-flags/qa-flag-environments.png)
+
+Click **Add Allocation** on the flag and serve the treatment or the experience to be tested to 100%.
+
+Set your flag to **On** to have it run in the selected environment.
+
+![Screenshot of allocation for Staging Environment](/img/guides/testing-feature-flags/qa-flag-staging-allocation.png)
+
+## Testing in Production
+
+To test in a Production environment, switch the environment on the flag to the **Production Environment**. 
+
+Decide what will indicate an internal user based on what information is available. Create a [Feature Gate](/feature-flagging/feature-gates#create-a-feature-gate) and set up a targeting condition that targets internal users. 
+
+In the example below, we target all users with an internal email address that ends in `@geteppo.com` or a specific user id. 
```

Comment:
Small nit here: I've seen some customers tell their engineers not to pass in PII info like email to our SDK. While we don't record that data, the event capture system will. It might be worth calling that out here in a note. Something like:

:::note
While Eppo does not store any raw data, note that targeting attributes will be passed into the logging callback function. If you do not want your logger to store user emails, you can apply the regex match before calling Eppo and then pass in a simple `is_internal_user` flag to the SDK call.
:::

I'm curious your thoughts, this might be overkill idk.